cli_test/cli_test.go

```diff
@@ -14,15 +14,17 @@ import (
 	"testing"
 	"time"
 
-	"github.com/tendermint/tendermint/crypto/ed25519"
-
 	"github.com/stretchr/testify/require"
 
+	"github.com/tendermint/tendermint/crypto/ed25519"
```

Comment:
What is this file for? It doesn't seem to run when I do `go test -mod=readonly ./cli_test -run=*` it says `no test files`.

Reply:
Those tests have to be run with a build tag "cli_test". I added a test-all command to the makefile to avoid having to remember everything. We should probably make the sims and the cli tests use the same format.